sympy/combinatorics/homomorphisms.py

```diff
@@ -487,21 +487,28 @@ def group_isomorphism(G, H):
 
         if not counterexample:
             # Returns a `GroupHomomorphism` object.
-            return homomorphism(G, H, G.generators, H, check=False)
+            return homomorphism(G, H, G.generators, subset, check=False)
 
     return None
 
-def is_isomorphic(G, H):
+def is_isomorphic(G, H, isomorphism=False):
     '''
     Check if the given groups are isomorphic.
 
     Arguments:
         G (a finite `FpGroup` or a `PermutationGroup`) -- First group
         H (a finite `FpGroup` or a `PermutationGroup`) -- Second group
+        isomorphism(boolean) -- This computes the isomorphism when set to True
 
     Returns:
-    `True` if the groups are isomorphic
-    `False` if the groups are not isomorphic
+    if isomorphism is set to `False` --
+        `True` if the groups are isomorphic
+        `False` if the groups are not isomorphic
+    if isomorphism is set to `True` --
+        Returns a tuple (boolean, `GroupHomomorphism` object).
```

Comment:
You should include some examples to show how the function can be used both with and without `isomorphism=True`. Have a look at other functions' docstrings for how to format this section. I think `homomorphism` has one.

It would probably make sense to have an example of just `is_isomorphic`, one with `isomorphism=True` that returns `(False, None)` and one that returns `(True, T)` where `T` is an isomorphism (you could store these in `(check, T)` and apply `T` to some element to demonstrate it's a homomorphism object).

Reply:
Okay 👍